src/main/resources/data/ars_nouveau/patchouli_books/worn_notebook/zh_cn/entries/machines/potion_jar.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "药水罐",
+    "icon": "ars_nouveau:potion_jar",
+    "category": "machines",
+    "pages": [
+        {
+            "type": "text",
+            "text": "一个最多能存储 100 份药水的罐子。药水可以使用空玻璃瓶或者药水烧瓶舀出。薇克精在进行药水自动合成时会使用这种罐。潜行时对药水罐使用支配之杖可以锁定药水的种类。锁定种类后的药水罐只会从薇克精处接收同种药水。药水罐可以被比较器测量。"
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

这种情况下，可以自行添加主语，例如：

```suggestion
            "text": "一个最多能存储 100 份药水的罐子。你可以使用空玻璃瓶或者药水烧瓶舀出其中的药水。薇克精在进行药水自动合成时会使用到这种罐。潜行时对药水罐使用支配之杖可以锁定药水的种类。锁定种类后的药水罐只会从薇克精处接收同种药水。药水罐可以被比较器测量。"
```
这种主语添加一般运用于被动句式中。被动句式往往是暗藏了某个主体作为行为发出者，而在中文语境下，被动句式的使用频率会比主动句式低很多，并且有时候英文的被动句式翻译为中文，会显得略微奇怪。可以看下面的这个句子，出自现在都还没有改正的 禁忌与奥秘 的指导书的提示文本：

> [这本《禁忌之书》很快就能读懂了..](https://github.com/stal111/Forbidden-Arcanus/blob/52560ffae76e9ba90fe9e43432abcfbcd874e96f/src/main/resources/assets/forbidden_arcanus/lang/zh_cn.json#L264)

这句话首先是有语病的，修正之后，为：

> 这本《禁忌之书》很快就能被读懂了..

但仍显奇怪，因为我们一般不这么说。换成如下句子就好很多：

> 你很快就能读懂这本《禁忌之书》了..

这就是一个典型的被动转主动的案例。

---

我改改

---

正常，我提倡无论是谁交翻译PR最好是都叫人来 review。

---

改完了